protos/attestation.proto

```diff
@@ -8,6 +8,7 @@ enum Tee {
     SNP = 2;
     TDX = 3;
     Sample = 4;
+    CSV = 6;
```

Comment:
is `6` as expected?

Reply:
This is because: https://github.com/confidential-containers/kbs/blob/main/proto/attestation.proto#L5

Reply:
Yes, AzSnpVtpm is not added to AS, but to kbs.